android/src/main/java/com/apptentive/android/sdk/reactlibrary/RNApptentiveModule.java

```diff
@@ -93,6 +109,26 @@ public void register(ReadableMap configuration, Promise promise) {
 		}
 	}
 
+	private ApptentiveLog.Level parseLogLevel(String logLevel) {
+		if (!StringUtils.isNullOrEmpty(logLevel)) {
+			switch (logLevel.toLowerCase()) {
+				case "verbose":
+					return ApptentiveLog.Level.VERBOSE;
+				case "debug":
+					return ApptentiveLog.Level.DEBUG;
+				case "info":
+					return ApptentiveLog.Level.INFO;
+				case "warn":
+				case "warning":
+					return ApptentiveLog.Level.WARN;
+				case "error":
+					return ApptentiveLog.Level.ERROR;
```

Comment:
There is also `ASSERT` and DEFAULT`. They are not needed here?

Reply:
This is Android specific and I'm not expecting people to use them in JavaScript code